### PR TITLE
Rate Limiting for entire website

### PR DESCRIPTION
**Pull request by ATHARVA-GAWAS**
Fixes issue

#1708 #888

### CODE REVIEW

File: website/tests.py [hunk 1]
```diff
@@ -131,3 +135,16 @@ def test_on_hide(self):
 
             if "hidden" not in filename:
                 self.assertFalse(True, "files rename failed")
+
+class IssueBaseCreateTest(APITestCase):
+    def test_throttle_exceed_limit(self):
+        for i in range(10000):
+            data = {'url': 'http://example.com', 'description': 'test', 'markdownInput': 'test', 'screenshots': 'test', 'captcha_1': 'PASSED'}
+            response = self.client.post('/report/', data)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (DonnieBLT) [inline website/tests.py:144]: Is this supposed to be a 200?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
See comment @ATHARVA-GAWAS

File: website/tests.py (hunk 1 above)
Comment (ATHARVA-GAWAS) [reply website/tests.py:144]: Let me check again

Comment (ATHARVA-GAWAS) [reply website/tests.py:144]: Since the test if for whether the form is valid or not

Comment (DonnieBLT) [reply website/tests.py:144]: The test should be if the rate limit was hit

**Pull request closed without merging**